src/widlprocess.ts

```diff
@@ -130,8 +130,19 @@ function convertInterfaceCommon(i: webidl2.InterfaceType | webidl2.InterfaceMixi
             addComments(result.constants!.constant[member.name], commentMap, i.name, member.name);
         }
         else if (member.type === "attribute") {
-            result.properties!.property[member.name] = convertAttribute(member, result.exposed);
-            addComments(result.properties!.property[member.name], commentMap, i.name, member.name);
+            const { properties } = result;
+            const prop = convertAttribute(member, result.exposed);
+            addComments(prop, commentMap, i.name, member.name);
+
+            if (member.name in properties!.namesakes!) {
+                properties!.namesakes![member.name].push(prop);
```

Comment:
it looks like this can happen more than once, but the code in mergeNamesakes assumes there's only one entry in each `namesakes` in line 20: `for (const [prop] of ...`. Is that correct?

Reply:
Yup, because in line 20 the `namesakes` value is already filtered by exposure (`[Exposed]` in IDL).